text/0000-landing-page.md

```diff
@@ -0,0 +1,53 @@
+# Meta
+[meta]: #meta
+- Name: Create a community landing page
+- Start Date: 1-6-2020
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+People interested in the project don't have a central place to go for information. This RFC proposes the creation of a community or "landing page" repo at `buildpacks/buildpacks` with high level information about the project including key personel, governance, guidelines for contributions, meeting schedules, etc.
```

Comment:
This should. more clearly define "people".  I believe we want to narrow this down to "prospective contributors" as we already have https://buildpacks.io for "prospective users".

Reply:
I'm okay with making that distinction. In my mind a prospective user is often also a prospective contributor and vice-versa, but for clarity and limiting the scope of this its seem wise to target the contributors.

Reply:
Added language that clarifies that this is targeting the contributor community